static/app/utils/replays/hooks/useDeadRageSelectors.tsx

```diff
@@ -6,24 +6,33 @@ import {
   DeadRageSelectorQueryParams,
 } from 'sentry/views/replays/types';
 
-export default function useRageDeadSelectors(
-  params: DeadRageSelectorQueryParams = {per_page: 10, sort: '-count_dead_clicks'}
-) {
+export default function useRageDeadSelectors(params: DeadRageSelectorQueryParams) {
   const organization = useOrganization();
   const location = useLocation();
   const {query} = location;
 
-  return useApiQuery<DeadRageSelectorListResponse>(
-    [
-      `/organizations/${organization.slug}/replay-selectors/`,
-      {
-        query: {
-          ...query,
-          per_page: params.per_page,
-          sort: params.sort,
+  const {isLoading, isError, data, getResponseHeader} =
+    useApiQuery<DeadRageSelectorListResponse>(
+      [
+        `/organizations/${organization.slug}/replay-selectors/`,
+        {
+          query: {
```

Comment:
these properties are currently preserved across page changes (e.g. from /replays/ -> /rage-clicks/ -> /dead-clicks/... whatever filters you set will be kept each time). need to fix

Reply:
keeping statsPeriod/project/env sounds like a good thing!